client/apps/edit/components/message/test/index.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { MessageModal } from '../index'
+import { ModalTypes } from '../modalTypes'
+import { shallow } from 'enzyme'
+
+describe('MessageModal', () => {
+  it('renders lock modal', () => {
+    const type = 'locked'
+    const data = ModalTypes[type]
+    const component = shallow(<MessageModal {...getProps(type)} />)
```

Comment:
This is perhaps arguable, but `mount` will always provide more real-world assurances than `shallow` while also being less confusing down the line (when some future dev wants to assert the existence of some random child in the tree but length always unexpectedly returns 0, or events don't bind / trigger, etc). That said, `dive` is really nice / useful.

Reply:
Right, I didn't think this test warranted the use of `mount` but that may change if we go deeper down the render tree